ONSdigital/rm-survey-service
# Dependabot alerts

# What and why?
Updating packages to fix Dependabot alerts 
# How to test?
Deploy to your environment and test that the response ops survey pages are still working, an acceptance tests run should do this as well
# Trello
https://trello.com/c/FA2i2kMw/1652-rm-survey-service-security-alerts
